pages/index.js

```diff
@@ -15,10 +17,8 @@ export default class Index extends React.Component {
             {"name": "keywords", "content": "sample, something"},
           ]}
```

Comment:
It does render stuff, check what `Helmet` is [here](https://github.com/nfl/react-helmet)